src/Socialment.php

```diff
@@ -4,4 +4,21 @@
 
 class Socialment
 {
+    protected array $providers = [];
+
+    public function getProvider($driver)
+    {
+        return $this->providers[$driver];
```

Comment:
This has been added by another PR in a slightly more enhanced fashion. Please pull and revise.